edgePy/DGEList.py

```diff
@@ -26,10 +27,12 @@ class DGEList(object):
         samples: Array of sample names, same length as ncol(counts).
         genes: Array of gene names, same length as nrow(counts).
         norm_factors: Weighting factors for each sample.
-        group: ...
+        groups_in_list: a list of groups to which each sample belongs, in the same order as samples *or*
+        groups_in_dict: a dictionary of groups, containing sample names.
         to_remove_zeroes: To remove genes with zero counts for all samples.
         filename: a shortcut to import NPZ (zipped numpy format) files.
-
+        current_type:  None means raw counts, otherwise, if transformed, store a string (eg. 'cpm', 'rpkm', etc)
```

Comment:
current_type is not descriptive.

- normalization_type or
- current_normalization_type

could be it?

Reply:
I would also rather speak about a state that a type

Reply:
Please feel free to make a suggestion!

Renamed.